entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Getting Started - EF Core
+author: masoucou
+ms.date: 07/07/2020
+uid: core/get-started/xamarin
+---
+
+# Getting Started with EF Core and Xamarin
+
+In this tutorial, you create a Xamarin.Forms application that performs data access against a SQLite database using Entity Framework Core.
+
+You can follow the tutorial by using Visual Studio on Windows or Visual Studio for Mac.
+
+[View this article's sample on GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/GetStarted).
+
+## Prerequisites
+
+Install one of the below:
+
+* [Visual Studio 2019 version 16.3 or later](https://www.visualstudio.com/downloads/) with this  workload:
+  * **Mobile Development with .NET**
+* [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/)
+
+This [documentation provides detailed step-by-step installation instructions](https://docs.microsoft.com/xamarin/get-started/installation) for each platform.
+
+## Download and run the sample project
+
+To run and explore this sample application, download the code on GitHub.
+
+Once downloaded, open the solution file `EFGettingStarted.sln` in Visual Studio or Visual Studio for Mac and run the application on the platform of your choice.
+
+When the app first starts, it will populate the local SQLite database with 2 entries representing blogs.
+
+Click the **Add** button in the toolbar.
+
+A new page will appear that allows you to enter information about a new blog. 
+
+Fill out all the info and click **Save** from the toolbar. The new blog will save to the app's SQLite database and will show in the list.
+
+You can click on one of the blog entries in the list and see any posts for that blog.
```

Comment:
Would it be possible to get a screenshot to show?

Reply:
Yes! Screenshots are coming up. I tagged you a bit too early for review. 😄

Reply:
Screenshots in latest commit.